packages/api/controllers/nomination.js

```diff
@@ -77,13 +77,44 @@ const createNomination = async (req, res) => {
     return res.status(500).json({ error: error.message });
   }
 };
+
+const resendEmail = async (req, res) => {
+  const { id } = req.params;
+  const { recipient, emailType } = req.body;
+  try {
+    const nomination = await db.Nomination.findOne({
+      where: { id },
+    });
+    if (recipient === 'family-member' && emailType === 'hipaa') {
```

Comment:
Can probably simplify these if statements with something like this: ``console.log(`email sent to: ${recipient.replace('-', ' ')}, email type sent: ${emailType}`);``

Reply:
fixed!